### DISCUSSION TITLE
Pro server restore / migration appears to not have worked.

### DISCUSSION THREAD

**Discussion by Jono-Moss** [rustdesk/rustdesk#8149] [Q&A] [answered]
I want to migrate to a different VPS server. 
I used the Backup/Restore method found on this page:
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/installscript/

The restore said that it was successful, however when I go to the RustDesk Console.
My credentials do not work, I had to use the default Login details:
admin
test1234

In order to login and I see that non of the devices have been restored.
So it appears that the restore did not work.
I need to make sure the migration and all data is restored correctly first before I can move the license.

can someone help assist with this.

**Comment by rustdesk** [comment, marked as answer]
i suggest you copy dababase files to new server manually

https://github.com/rustdesk/rustdesk-server-pro/discussions/184

**Reply by Jono-Moss** [reply]
Thank you.
